drivers/camera/himax.c

```diff
@@ -37,7 +37,7 @@
 
 #include "rt/rt_api.h"
 
-//#define ACTIVATE_I2C 1
+#define ACTIVATE_I2C 1
```

Comment:
If we activate it, this will break the camera driver on RTL platform and gvsoc. Could you replace it with a runtime check of the platform to activate it only on the board and the fpga ?

Reply:
OK, sorry for that, I will change it as you said. One question of that, for fpga we have "ARCHI_PLATFORM_FPGA", and for the board, we use the same? or we have other flag for the boards?